rollup.config.js

```diff
@@ -1,8 +1,59 @@
-export default {
+
+import builtins from 'rollup-plugin-node-builtins'
+import replace from 'rollup-plugin-re'
+import resolve from 'rollup-plugin-node-resolve'
+import commonjs from 'rollup-plugin-commonjs'
+import json from 'rollup-plugin-json'
+
+export default [{
+    input: 'src/index.js',
+    name: 'ocapi-client',
+    output: {
+        file: 'dist/commercecloud-ocapi-client.js',
+        format: 'umd'
+    },
+    plugins: [
+        json(),
+        /*
+         * NOTE: Do replace before commonjs
+         * This is a hack for rollup
+         * In current rollupJS version, it does not handle reassigned require function
+         * in certain packages that uses commonjs helper
+         * We need to manually remove the hijacked require function
+         * For more info: https://github.com/rollup/rollup-plugin-commonjs/issues/166#issuecomment-328853157
+         */
+        replace({
+            patterns: [{
+                match: /formidable(\/|\\)lib/,
+                test: 'if (global.GENTLY) require = GENTLY.hijack(require);',
```

Comment:
@kevinxh It looks like this library has something to do with testing. Because we don't require testing for building the UMD it may be possible to set GENTLY to false and not have require be reassigned. We'd then not have to used this replace plugin and clean things up a little bit.